pkgs/applications/misc/crane-gps-watch/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, libxml2 }:
+
+stdenv.mkDerivation rec {
+  name = "crane-gps-watch-${version}";
+  version = "221f99dc2925a07658f7b6e01789ced1a18cc08c";
```

Comment:
Versions ought to satisfy `new > old`, so ISO 8601 dates are a better choice for vcs checkouts.